mozilla-release/browser/components/preferences/in-content/security.js

```diff
@@ -457,6 +483,32 @@ var gPrivacyManagers = {
       "sourceURI": "https://s3.amazonaws.com/cdncliqz/update/browser/firefox@ghostery.com/latest.xpi"
     }];
   },
+}
+
+function ItemHandler(container, addonDescriptor, addonObj, status) {
+  this._container = container;
+  this._desc = addonDescriptor;
+  this._addon = addonObj;
+  if (status == 'new') {
+    this._listItem = this.createItem(addonDescriptor, status);
+  } else { // installed
+    this._listItem = this.createItem(addonObj, status);
+  }
+  this._listItem.addEventListener("installClicked", function(e) {
+    this.onInstallClick();
+  }.bind(this))
+  this._listItem.addEventListener("unInstallClicked", function(e) {
+    this.onUninstallClick();
+  }.bind(this))
```

Comment:
this._listItem.addEventListener("unInstallClicked", this.onUninstallClick.bind(this));